### ISSUE TITLE
Support more mouse buttons

### ISSUE THREAD

**Issue by Dagon** [UltimateHackingKeyboard/agent#834]
My mouse has 7 buttons - it would be nice to be able to send buttons higher than 3 from the mouse layer on the keyboard.

**Comment by mondalaci** [comment]
We'll make this happen. I slightly edited the subject of this issue because mouse actions can be assigned to any layers, not just the mouse layer.

**Comment by ert78gb** [comment]
@mondalaci I fixed the multi-button selection and added the Key Action texts.
The question is how to visualize the action on the keyboard?
The left-click visualization
<img width="90" alt="Screenshot 2019-08-21 at 20 40 50" src="https://user-images.githubusercontent.com/496775/63459485-e7fe2400-c454-11e9-80d6-fb8540a7edab.png">
 
Now I just added the number as the text.
<img width="64" alt="Screenshot 2019-08-21 at 20 47 15" src="https://user-images.githubusercontent.com/496775/63459558-09f7a680-c455-11e9-97de-afb3d85180cd.png">
